### PR TITLE
docs: Fix misspelling in CI documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR fixes a misspelling in CI documentation readme.

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ them to merely debug issues.
 
 In the previous section we've mentioned using different runners, now in this section we'll go through each type of runner used.
 
-- Cost free runners:  Those are the runners provided by GIthub itself, and
-  those are fairly small machines with no virtualization capabilities enabled - 
+- Cost free runners:  Those are the runners provided by Github itself, and
+  those are fairly small machines with no virtualization capabilities enabled.
```
Comment (fidencio) [inline ci/README.md:81]: Well, as you're already touching this, I think it's worth updating the documentation as the Cost free runners now support virtualization.

**Review by fidencio — CHANGES_REQUESTED** [review]
@GabyCT, as you're already touching it, it's worth fixing not only the mispelling but also updating the line with correct information.

File: ci/README.md (hunk 1 above)
Comment (GabyCT) [reply ci/README.md]: @fidencio changes applied

**Review by fidencio — APPROVED** [review]
lgtm, thanks @GabyCT!

**Review by dborquez — APPROVED** [review]
LGTM, thank you @GabyCT

**Pull request merged**